geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/view/RoutedDataStoreView.scala

```diff
@@ -0,0 +1,135 @@
+/***********************************************************************
+ * Copyright (c) 2013-2019 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.index.view
+
+import org.geotools.data._
+import org.geotools.data.simple.{EmptySimpleFeatureReader, SimpleFeatureSource}
+import org.locationtech.geomesa.curve.TimePeriod.TimePeriod
+import org.locationtech.geomesa.index.stats.GeoMesaStats.GeoMesaStatWriter
+import org.locationtech.geomesa.index.stats.RunnableStats.UnoptimizedRunnableStats
+import org.locationtech.geomesa.index.stats.{GeoMesaStats, HasGeoMesaStats, NoopStatWriter}
+import org.locationtech.geomesa.index.view.RoutedDataStoreView.RoutedStats
+import org.locationtech.geomesa.utils.io.CloseWithLogging
+import org.locationtech.geomesa.utils.stats._
+import org.opengis.feature.`type`.Name
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+import org.opengis.filter.Filter
+
+/**
+  * Routed querying against multiple data stores
+  *
+  * @param stores delegate stores
+  * @param router query router
+  * @param namespace schema namespace
+  */
+class RoutedDataStoreView(val stores: Seq[DataStore], val router: RouteSelector, namespace: Option[String])
+    extends MergedDataStoreSchemas(stores, namespace) with HasGeoMesaStats {
```

Comment:
Ok, what's the schema namespace?

Reply:
that's just the namespace configured via geoserver, used to namespace simple feature types in getSchema